benefits/core/templates/core/agency-index.html

```diff
@@ -2,10 +2,10 @@
 {% load i18n %}
 
 {% block title %}
-  {% translate "core.pages.agency_index.title" %}
+  {% translate "Introduction" %}
 {% endblock title %}
 
 {% block call-to-action %}
   {% url "eligibility:index" as url_continue %}
-  <a href="{{ url_continue }}" class="btn btn-lg btn-primary">{% translate "core.pages.index.continue" %}</a>
+  <a href="{{ url_continue }}" class="btn btn-lg btn-primary">{% translate "Get Started" %}</a>
```

Comment:
I don't know why but my agency index shows up like this
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/18cf3cfa-5746-4831-8e1d-e6940680d4fa">
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/3da937de-d9d3-481f-ba5e-719df515f5de">

Reply:
Yikes, good catch, I'm seeing this too. Will fix.

Reply:
Doh, had a duplicate `msgstr` with that text, removed in [ac122bc](https://github.com/cal-itp/benefits/pull/1626/commits/ac122bc65497ba1b702a59517d50d46b6fa48833)

Reply:
fix confirmed